packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js

```diff
@@ -83,8 +88,7 @@ class LeftMenuLinkContainer extends React.Component { // eslint-disable-line rea
 }
```

Comment:
Maybe something like this would be a shorter version :

`{_.map(this.links, (link, key) => <LeftMenuLink key={key} icon={link.icon} label={messages[link.label].id} destination={`/${_.kebabCase(link.label)`} /> )}`